sigh/lib/sigh/resign.rb

```diff
@@ -121,7 +124,7 @@ def installed_identies
       ids = []
       available.split("\n").each do |current|
         begin
-          (ids << current.match(/.*\"(.*)\"/)[1])
+          (ids << "#{current.match(/[a-zA-Z0-9]{40}/)}#{ID_SEPARATOR}#{current.match(/.*\"(.*)\"/)[1]}")
```

Comment:
@somedev I think that sounds like a good plan! I'd be happy to review that PR if either of you submit it! :+1: